zerver/views/webhooks/codeship.py

```diff
@@ -0,0 +1,47 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+import ujson
+
+
+CODESHIP_SUBJECT_TEMPLATE = '{project_name} build status.'
```

Comment:
We usually don't use periods at the end of topics in messages, can you fix that?  Also maybe worth cutting off `build status` from the subject (and just having it be project_name), since it seems like it's implied by the fact this is being sent by the codeship integration.  If there were more types of messages codeship sends it'd be worth thinking about having "project build" and "project deploy" messages, but it seems like all it does is builds.